packages/carousel/src/ArrowController.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+// import PropTypes from 'prop-types'
```

Comment:
Commented this out for now.  Was getting a warning, which thwarted storybook:

```
"export 'default' (imported as 'PropTypes') was not found in 'prop-types'
```

Anyone see this before?

Reply:
This is probably related to what #406 is attempting to resolve